README.md

```diff
@@ -106,18 +107,19 @@ var options = {}
 
 // auth is required
 // it can either be a username & password...
-var auth = {
+options.auth = {
   username: 'ngoldman',
   password: 'XXXXXXXX'
 }
+
 // or an API token
-var auth = {
+options.auth = {
   token: 'XXXXXXXX'
 }
 
-ghRelease(options, auth, function (err, result) {
+ghRelease(options, function (err, result) {
   if (err) throw err
-  console.log('Release URL: ' + result)
+  console.log(result) // create release response: https://developer.github.com/v3/repos/releases/#response-4
```

Comment:
@paulcpederson did you actually change the content of the result param? it should return the release URL, not the raw API release response.